frontend/pages/queries/live/LiveQueryPage/LiveQueryPage.tsx

```diff
@@ -143,30 +145,34 @@ const RunQueryPage = ({
 
   useEffect(() => {
     detectIsFleetQueryRunnable();
-    if (!queryId) {
-      setLastEditedQueryId(DEFAULT_QUERY.id);
-      setLastEditedQueryName(DEFAULT_QUERY.name);
-      setLastEditedQueryDescription(DEFAULT_QUERY.description);
-      setLastEditedQueryBody(DEFAULT_QUERY.query);
-      setLastEditedQueryObserverCanRun(DEFAULT_QUERY.observer_can_run);
-      setLastEditedQueryFrequency(DEFAULT_QUERY.interval);
-      setLastEditedQueryLoggingType(DEFAULT_QUERY.logging);
-      setLastEditedQueryMinOsqueryVersion(DEFAULT_QUERY.min_osquery_version);
-      setLastEditedQueryPlatforms(DEFAULT_QUERY.platform);
-    }
   }, [queryId]);
 
+  useEffect(() => {
```

Comment:
Let's leave a TODO to try to remove the local state and this effect in a future iteration.